examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -215,6 +225,133 @@ async function example_API_SetIfNotExists(cacheClient: CacheClient) {
   }
 }
 
+async function example_API_ListFetch(cacheClient: CacheClient) {
+  const result = await cacheClient.listFetch('test-cache', 'test-list');
+  if (result instanceof CacheListFetch.Hit) {
+    console.log(`List fetched successfully: ${result.toString()}`);
```

Comment:
For all of these APIs where the response object has a value accessor, I would like to consume it in the log statement so we're showing people what the common method is that they are going to want to call.  In this class I think it's `valueListString`?